blog/_posts/2016-10-08-team-l1ghtsab3r-signing-off.md

```diff
@@ -0,0 +1,74 @@
+---
+title: “Team l1ghtsab3r signing off...”
+layout: post
+created_at: Sat Oct 08 2016
```

Comment:
Could you please change the date here to Mon Oct 10 2016, as that's when we will be publishing it? Make sure to also update both the file name and the permalink so that both have the new date (2016-10-10).